src/components/OperatorArea/index.tsx

```diff
@@ -22,11 +22,9 @@ export function OperatorArea({ assembly, metadata }: OperatorAreaProps) {
   const details: ReactNode[] = useMemo(() => {
     if (!metadata) return [];
 
-    const {
-      date,
-      publisher: { username },
-      links: { repository },
-    } = metadata;
+    const { date } = metadata;
+    const username = assembly?.author.name;
```

Comment:
is this `date` field present on `assembly.metadata`? If so we can cut out the metadata.json request and just read from the assembly response

Reply:
Nope, this is the date the package was published to npm, so not available on the assembly (which is created during package build)